devel/kube-linter/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/stackrox/kube-linter 0.2.2
```

Comment:
Let's add a `revision` field, but otherwise looks good to merge.

Reply:
Thanks, done!